packages/dapp/components/lib/utils.ts

```diff
@@ -31,10 +31,10 @@ export async function performTransaction(transaction: Promise<ContractTransactio
 }
 
 export async function erc1155BalanceOf(addr: string, erc1155UbiquityCtr: Contract): Promise<BigNumber> {
-  const treasuryIds = await erc1155UbiquityCtr.holderTokens(addr);
+  const treasuryIds = await erc1155UbiquityCtr!.holderTokens(addr);
 
   const balanceOfs = treasuryIds.map((id: string) => {
-    return erc1155UbiquityCtr.balanceOf(addr, id);
+    return erc1155UbiquityCtr!.balanceOf(addr, id);
```

Comment:
These ! Should be forbidden. The typescript template settings should be updated on this repo @rndquu

Reply:
> These ! Should be forbidden. The typescript template settings should be updated on this repo @rndquu

Yeah, non-null assertion operator in `erc1155BalanceOf` function should be removed.
I have applied non-null assertion operator to protocol contracts in order to avoid object null checking.. but protocol contracts can't be null.. so it will not generate any errors

Reply:
use `?` instead